Insolvency.CalculationsEngine.Redundancy.BL/Services/Implementations/HolidayCalculationService.cs

```diff
@@ -29,11 +30,29 @@ public async Task<HolidayCalculationResponseDTO> PerformHolidayCalculationAsync(
             if (data.Hpa != null)
                 return await PerformHolidayCalculationWithHPAAsync(data, options);
             else if (data.Htnp != null && data.Htnp.Any())
-                return await PerformHolidayCalculationWithoutHPAAsync(data, options);
+                return await PerformHolidayCalculationWithoutHPAAsync(data.Htnp, options);
```

Comment:
What is the reasoning behind the change to use Htnp?
Is this a change to existing code?

Reply:
To reuse this piece of code in PerformIrregularHolidayCalculationAsync method for calculating.